project-base/templates/Front/Content/Product/detail.html.twig

```diff
@@ -146,7 +146,7 @@
                     </div>
                 </div>
 
-                {% if product.isMainVariant %}
+                {% if product.isMainVariant and not product.calculatedSellingDenied %}
```

Comment:
I don't see context with user story. Is this condition ok? Or I need details :smile:

Reply:
This condition is : when main variant is disabled to sell... => dont print table header - bcs variants are off.

Reply:
Aaa, point three in US :smile:. Thank you